node/packages/blockbook/src/index.ts

```diff
@@ -16,3 +17,28 @@ export interface WebsocketRepsonse {
   id: string
   data: NewBlock | Tx | SubscriptionResponse
 }
+
+export function getAddresses(tx: Tx): Array<string> {
```

Comment:
a comment here might make it clearer what this function does.

From reading the code I see its getting all addresses involved in a tx but somebody without crypto experience might wonder wtf is going on. or maybe a different name would suffice?